ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/[externalID]/ResyncModal.tsx

```diff
@@ -68,45 +69,53 @@ export default function ResyncModal({ isOpen, onClose, url }: Props) {
       }
     >
       <div className="border-b border-slate-200 px-6">
+        {platform === 'vercel' && !failure && (
+          <Alert className="my-2" severity="info" showIcon={false}>
```

Comment:
suggestion: The top spacing looks a little cramped, should the spacing around this be consistent? Not sure what the exact spacing should be. Please ignore if you already covered this with John!
```suggestion
          <Alert className="my-6" severity="info" showIcon={false}>
```

![image](https://github.com/inngest/inngest/assets/1509457/965d2b4c-a176-4beb-ae17-e2760db4d1ff)